internal/operators/cnv/utils.go

```diff
@@ -0,0 +1,1855 @@
+package cnv
+
+import (
+	"bytes"
+	"encoding/json"
+	"html/template"
+
+	models "github.com/openshift/assisted-service/models"
+)
+
+func setOperatorStatus(cluster *models.Cluster, status string) error {
+	var operators models.Operators
+	err := json.Unmarshal([]byte(cluster.Operators), &operators)
+	if err != nil {
+		return err
+	}
+	for _, operator := range operators {
+		if operator.OperatorType == models.OperatorTypeCnv {
+			operator.Status = status
+			break
+		}
+	}
+	clusterOperators, err := json.Marshal(operators)
+	if err != nil {
+		return err
+	}
+
+	cluster.Operators = string(clusterOperators)
+	return nil
+}
+
+// Manifests returns manifests needed to deploy CNV
+func Manifests(OpenShiftVersion string) (map[string]string, error) {
+	manifests := make(map[string]string)
+	manifests["99_openshift-cnv_crd.yaml"] = cnvCRD
+	manifests["99_openshift-cnv_ns.yaml"] = cnvNamespace
+	cnvSubs, err := cnvSubscription(OpenShiftVersion)
+	if err != nil {
+		return map[string]string{}, err
+	}
+	manifests["99_openshift-cnv_subscription.yaml"] = cnvSubs
+	manifests["99_openshift-cnv_operator_group.yaml"] = cnvGroup
+	manifests["99_openshift-cnv_hco.yaml"] = cnvHCO
+	return manifests, nil
+}
+
+func cnvSubscription(OpenShiftVersion string) (string, error) {
+	data := map[string]string{
+		"OPENSHIFT_VERSION": OpenShiftVersion,
+	}
+
+	const cnvSubscription = `apiVersion: operators.coreos.com/v1alpha1
```

Comment:
WDYT about promoting this const definition to a file-level one?

Reply:
:+1: